gcc/config/aarch64/aarch64-coff.h

```diff
@@ -55,7 +55,7 @@
     {							\
       fprintf (STREAM, "\t.comm\t");			\
       assemble_name (STREAM, NAME);			\
-      asm_fprintf (STREAM, ", %d, %d\n", 		\
+      asm_fprintf (STREAM, ", %d\t%@ %d\n", 		\
```

Comment:
please update PR branch, it is reverting changes

Reply:
Thank you for noticing. I've just haven't finished that step in between context switches.